src/MFRC522.cpp

```diff
@@ -177,7 +179,10 @@ MFRC522::StatusCode MFRC522::PCD_CalculateCRC(	byte *data,		///< In: Pointer to
 			result[1] = PCD_ReadRegister(CRCResultRegH);
 			return STATUS_OK;
 		}
+		yield();
 	}
+	while (millis() < deadline);
```

Comment:
please use typecast for `millis` to `uint32_t`

Reply:
`millis` is `unsigned long` on the Arduino platform. Would you prefer `unsigned long` over `uint32_t`?

Reply:
i prefer the `uint32_t` over the generic `unsigned long`